index.html

```diff
@@ -79,40 +79,44 @@ <h2>Get to know me</h2>
                 </p>
             </article>
 
-            <div class="decorative__img__holder"></div>
+            <div class="decorative__img__holder">
+                <img src="/assets/img/coffeecan_on_stol.jpg" alt="modern coffee can standing och a wooden stol">
```

Comment:
Länkade jag redan denna om bilder? https://youtu.be/u_QRe0RX_6Q

Reply:
Tror inte det, ska kolla in det.